exs_numpy.py

```diff
@@ -0,0 +1,423 @@
+#Submitted by Keren Or Hadad 208025205
+
+import numpy as np

```

Comment:
In general its is better practice to organize your code in functions and methods. Your current program is not reusable.